src/main/webapp/js/user-page-loader.js

```diff
@@ -107,6 +105,19 @@ function buildMessageDiv(message) {
   return messageDiv;
 }
 
+function fetchImageUploadUrlAndShowForm() {
+  fetch('/image-upload-url')
+    .then((response) => {
+      return response.text();
+    })
+    .then((imageUploadUrl) => {
+      const messageForm = document.getElementById('message-form');
+      messageForm.action = imageUploadUrl;
+      messageForm.classList.remove('hidden');
+      document.getElementById('recipientInput').value = parameterUsername;
```

Comment:
This addition of a "recipient" field doesn't seem directly related to uploading images.  Should it be a separate PR?

Reply:
That's super fair. I'm a bit stuck on how to create a separate PR for this. Should I only add and commit this file on another branch and then make a new PR?